sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3382,11 +3382,9 @@ object GpuOverrides extends Logging {
         override def tagExprForGpu(): Unit =
```

Comment:
On line 3378 can we remove the STRUCT support? That should update the docs to make it clear that we only supp Map<STRING,STRING>

Reply:
Thank you! Changes updated.